### DISCUSSION TITLE
Links wont open to new tab if you put address in address field

### DISCUSSION THREAD

**Discussion by Pistolay** [onemen/TabMixPlus#300] [General]
Found a bug... I am not sure where to report this, I apologize if this is the wrong place, I just didn't see a place specific to bug reporting.

In preferences  -> events -> new tabs

At the top there is a pull down "load on new tab" giving options, under this is an address field for a custom address. I have found when I put anything in here regardless to the pull down selection, if there is anything placed in this box (such as www.google.com) then you cannot right click and open links. once I cleared this field link opening worked again.

I am on firefox 126.0, although I saw this in previous versions but hadn't figured  it out. Windows 11...

**Comment by onemen** [comment]
Tab mix Plus [version 1.20.0](https://github.com/onemen/TabMixPlus/releases/download/v1.20.0/tab_mix_plus-1.20.0.xpi) is out.

Verify that all your firefox-scripts are up-to-date 
https://github.com/onemen/TabMixPlus/files/14075743/fx-folder.zip
https://github.com/onemen/TabMixPlus/files/14075742/utils_extensions_and_scripts.zip

**Comment by Pistolay** [comment]
YUP Yup… ALL UP TO DATE ON EVERYTHING… I reinstalled the scripts even tho they are the same version just to make sure that wasn’t the issue. I am running the latest version of TabMix.. I had this problem before but I didn’t know what it was at the time, reset the config to default and it worked again, but with the new version of Firefox (126.0) they changed their home screen for new tabs, and I would always prefer that a new tab open to google.. so I set it to do so and I figured out this bug.

 
![Capture](https://github.com/onemen/TabMixPlus/assets/170133948/0caeb2b9-ebc4-4ffe-bb6a-cfb956f7c6e3)



 

If you put anything in the address box (circled above) you cannot open new tabs by right clicking a link and selecting “open link in new tab” or ”open link in foreground tab”… nothing happens. As soon as I clear it, then it works again.

 

I can’t be the only one having this issue.

 

Kinda funny that when opening a link to a new tab, you don’t need it to open to anything else first, just the address to the link, this setting should only come into play when opening a blank new tab…  

 

Pistolay

**Comment by onemen** [comment]
do you see any messages in the error console when you have this issue?

attach your Tab Mix Plus preferences.


the issue is probably caused by bad url.

try to use `https://www.google.com/` or `http://www.google.com/`

**Reply by Pistolay** [reply]
Okay, heres the error log.. and my config for tabmix. hope this helps.. This is not something new with the last firefox update, I had this problem before I just didn't know what was causing it. I cleared the config and all it started working again,  and didn't completely set the config due to being in a hurry. I don't know when it started (what version of FF). I jut checked, this is not an issue on waterfox.  

![image](https://github.com/onemen/TabMixPlus/assets/170133948/d4cb68c2-2a5b-49f6-8cc2-0198489f1993)

**Reply by Pistolay** [reply]
oh and as for the url, it doesnt work with any.... to be clear, its when you right click on a url and select to open in new tab. its not the url in the config. I have it set to www.google.com, although I am not opening a blank tab. when you open a blank new tab it works (with either the + for the file > open new tab).  I think a few others that have posted in this thread have slightly different issues.

**Reply by onemen** [reply]
@Pistolay 

Please, attach a file with your Tab Mix plus preferences, including the configuration to open a url in new tabs

**Reply by Pistolay** [reply]
Sorry, I thought I had uploaded it with my last post, sorry.

This is the non working config:
[TMPpref.txt](https://github.com/onemen/TabMixPlus/files/15409321/TMPpref.txt)

Here is the working config:
[TMPpref-good.txt](https://github.com/onemen/TabMixPlus/files/15409550/TMPpref-good.txt)

The only difference is the field has "www.google.com" in it, although I have tried a few other major ones with the same issue.

I did a little more troubleshooting, I have a few other addons that I used, so just to rule out that one of them may be somehow interfering with TMP I disabled them all, only running TMP and I still have the same behavior.

What I found is that as soon as I clear the field where it asks what url to open then it starts to work again, even if I don't change the pull-down option.


Non Working :
![image](https://github.com/onemen/TabMixPlus/assets/170133948/617b69fd-60d7-4937-b327-a006e2403d9f)

Working:
![image](https://github.com/onemen/TabMixPlus/assets/170133948/92a898b5-f9c7-440b-8fe3-169a8154937c)

**Reply by Pistolay** [reply]
OK.. I kept troubleshooting and got it to work:

If I put "https://www.google.com" it works fine.  AND "http://www.google.com" also works.

I have just been putting in the URL without the "http://" or "https://" prefix.

funny thing is that it works fine without the prefix if you are opening a new blank tab, but if you are opening a link in a new tab  then it has an an issue.

**Comment by Kalinette** [comment]
Personally with this version 1.20.0, the function: Force opening of links in a new tab no longer works at all on any site. I'm going back to the previous version. "Firefox 126.0"

**Comment by Kalinette** [comment]
I confirm. Downgrade to version 1.19.1 and everything works fine again.

**Comment by onemen** [comment]
do you see any messages in the error console when you have this issue?

attach your Tab Mix Plus preferences.

the issue is probably caused by bad url.

try to use https://www.google.com/ or http://www.google.com

**Comment by Kalinette** [comment]
I didn't try to understand since it no longer worked on any sites. Including here. I can not say more. Sorry.

**Comment by EDSln** [comment]
I too after upgrading to version 120 all links stopped opening in a new tab. Rolling back the version to 118 (I still have that version file) and upgrading back to 120 helped, now the links open in tabs as they should. The problem is not with the sites, before and after the problem they always opened in new tabs. Waterfox G6.0.14. But I saved the console output when the problem occurred.
![TMP](https://github.com/onemen/TabMixPlus/assets/34573155/8ef9ecd7-98a3-404d-a42d-3043c983058a)

**Comment by onemen** [comment]
@EDSln 

attach a file with your Tab Mix Plus preferences

**Reply by EDSln** [reply]
[TMPpref.txt](https://github.com/onemen/TabMixPlus/files/15366289/TMPpref.txt)

**Comment by onemen** [comment]
@EDSln, I don't see any issue with Tab Mix Plus version 1.20.0 on WaterFox G6.0.14

**Reply by EDSln** [reply]
Maybe the problem occurs when you first upgrade from an older version. I had the error on two profiles. But when I updated again, it is no longer there.

**Comment by onemen** [comment]
@EDSln,

sometimes the old version load from the profile cache instead of the new one.
https://onemen.github.io/tabmixplus-docs/other/installation/#profile-cache-issues
